### PR TITLE
Fix canjs/can-define#176. Related to #204

### LINKED ISSUE #204: The “source” link on the can-define/list/list doc pages doesn’t work
Steps to reproduce:
1. Go to the `can-define/list/list` docs: https://canjs.com/doc/can-define/list/list.html
2. Click the “source” link

Expected results: land on a page like this: https://github.com/canjs/can-define/blob/master/list/docs/define-list.md

Actual results: landed on this page: https://github.com/canjs/canjs/tree/v3.2.2/node_modules/can-define/list/docs/define-list.md

### PR DESCRIPTION
**Pull request by imaustink**
Fix bug breaking some `Edit on Github` links.
fixes canjs/can-define#176
fixes #204

### CODE REVIEW

File: templates/helpers.js [hunk 1]
```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return false;
             }
             var name = packageObject.name,
```
Comment (imaustink) [inline templates/helpers.js]: @leoj3n do you know what `packageObject.name` is? Is it intended to be the parent package name? So in this case `canjs`. Or is it meant to be the name of the current package being processed?

Comment (leoj3n) [inline templates/helpers.js]: It looks like it's the `docObject` or `docMap` in it's entirety.

https://github.com/bit-docs/bit-docs-tag-package/blob/master/package.js

https://github.com/canjs/bit-docs-html-canjs/blob/master/templates/title.mustache#L32

Comment (leoj3n) [inline templates/helpers.js]: Oh, I thought you were asking what is `packageObject`. For `packageObject.name`, that should simply be the `name` property, taken straight from `package.json`. So, for CanJS, it should be [just "`can`"](https://github.com/canjs/canjs/blob/master/package.json#L2).

**Comment by justinbmeyer** [comment]
This seems to be already working here: https://canjs.com/doc/can-define.html   Am I missing something?

File: templates/helpers.js [hunk 2]
```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
...
-                version = 'v' + packageObject.package.version,
-                srcPath = current.src.path.replace('node_modules/' + name + '/', ''),
+                srcPath = current.src.path.replace(SRC_PREFIX_REGEX, ''),
```
Comment (justinbmeyer) [inline templates/helpers.js]: Can you describe why this change was necessary?  What did `current.src.path` look like?

**Comment by imaustink** [comment]
@justinbmeyer try [this one](https://canjs.com/doc/can-define/list/list.prototype.filter.html) for a live example of a 404

File: templates/helpers.js (hunk 2 above)
Comment (imaustink) [inline templates/helpers.js]: The string looks like `node_modules/[module_name]/*`. The problem is `packageObject.name` is always `canjs`.

File: templates/helpers.js (hunk 1 above)
Comment (nlundquist) [inline templates/helpers.js]: afaik packageObject.name is actually the name given in the `@page` tag on the closest parent documentation page with a `@package` tag

**Comment by imaustink** [comment]
Closing in favor of #257
